NewsAPI/NewsAPI/VC/BaseNewsVC.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  BaseNewsVC.swift
+//  NewsAPI
+//
+//  Created by Katerina on 25/12/2023.
+//
+
+import Foundation
+import UIKit
+
+class BaseNewsVC: UIViewController {
+
+    func setupStyle() {
+        view.backgroundColor = .white
+    }
+    
+    func registerCell(collectionView: UICollectionView) {
```

Comment:
why do you need this method? 
i see you use it only once and in any other viewControllers you have theirs own registerCell methods. 
and naming, registerCells not Cell, you register 3 cells there.